docs/04.guides/04.cookbooks/32.Startup_Listeners/page.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Startup Listeners
+id: Startup Listeners-code
+---
+## Startup Listeners Code ##
+
+It have two kind of listeners. The first one is server.cfc that is called the server start, it exist at only once. Then second one is web.cfc that it can exists for every single web context on your server. We explain two listeners with a simple example below:
```

Comment:
***ToChange***
Lucee has two kinds of listeners. The first one is server.cfc that is called at server startup. It exists only once. The second listener is web.cfc which can exist for each web context on your server. We explain the two listeners with a simple example below: